nixos/modules/services/web-apps/peering-manager.nix

```
@@ -173,7 +193,10 @@ in {
           PEERINGDB_API_KEY = file.readline()
       '';
 
-      plugins = lib.mkIf cfg.enableLdap (ps: [ ps.django-auth-ldap ]);
+      plugins = lib.mkIf (cfg.enableLdap || cfg.enableOidc) (ps:
+        (lib.optionals cfg.enableLdap [ ps.django-auth-ldap ]) ++
+        (lib.optionals cfg.enableOidc (with ps; [ mozilla-django-oidc pyopenssl josepy ]))
+      );
```

```suggestion
      plugins = (ps:
        (lib.optionals cfg.enableLdap [ ps.django-auth-ldap ]) ++
        (lib.optionals cfg.enableOidc (with ps; [ mozilla-django-oidc pyopenssl josepy ]))
      );
```
there should be no drawback when returning an empty list